aws/logs_monitoring/lambda_function.py

```diff
@@ -465,6 +484,35 @@ def split(events):
     return metrics, logs
 
 
+# should only be called when INCLUDE_AT_MATCH and/or EXCLUDE_AT_MATCH exist
+def filter_logs(logs):
```

Comment:
would be nice that this method does not have to transform the logs to json, can we do it upstream please ?

Reply:
@ajacquemot Sure! Where do you think is a good place upstream to do it?

Reply:
Moved it to the `split` method